src/git.ts

```diff
@@ -31,9 +31,9 @@ export async function getCurrentGitBranch () {
   return r
 }
 
-export async function getGitDiff (from, to): Promise<RawGitCommit[]> {
+export async function getGitDiff (from: string | undefined, to: string): Promise<RawGitCommit[]> {
```

Comment:
Since i guess `HEAD` is the common use case for when omitting `from`, defaulting would make usage simpler:

```suggestion
export async function getGitDiff (from: string | undefined, to: string = 'HEAD'): Promise<RawGitCommit[]> {
```